azure-pipelines.yml

```diff
@@ -528,7 +528,7 @@ jobs:
       mkdir -p "$OBJ_DIR/hw"
       mkdir -p "$BIN_DIR/hw/top_englishbreakfast"
 
-      BOOTROM_VMEM="$BIN_DIR/sw/device/boot_rom/boot_rom_fpga_nexysvideo.scr.39.vmem"
+      BOOTROM_VMEM="$BIN_DIR/sw/device/boot_rom/boot_rom_fpga_nexysvideo.32.vmem"
```

Comment:
What happens to the ECC integrity in this case? 
We probably still have to add this since otherwise this may lead to bus integrity failures...

Reply:
Yeah, I was being lazy because Pirmin said it was already broken at runtime :-) But I've sorted it out properly now (generating ECC bits on the fly in the adapter).